goodfirstissue/slack/slack.go

```diff
@@ -0,0 +1,56 @@
+package slack
+
+import (
+	"io/ioutil"
+
+	"github.com/bluele/slack"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+var credentialsFile = "/var/openfaas/secrets/slack.yaml"
+
+//Tokens is for slack tokens
+type Tokens struct {
+	Token string `yaml:"token"`
+}
+
+//Client is slack client
+type Client struct {
+	slack *slack.Slack
+}
+
+//NewClient returns new twitter client
+func NewClient() (*Client, error) {
```

Comment:
Given that the function isn't guaranteed to be persistent, make sure you don't establish any websockets. I post messages to Slack simply by using a HTTP post and the appropriate incoming webhook URL. 

You are also tweeting, so IFTTT can be used to forward the tweets to Slack as an alternative.